QueueMessageSender/QueueMessageSender/Logic/Models/UserModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace QueueMessageSender.Logic.Models
+{
+    public class UserModel
+    {
+        [Key]
+        public int Id { get; set; }
+        public string Username { get; set; }
+        public string Password { get; set; }
+        public string RefreshToken { get; set; }
```

Comment:
In current implementatino user may have only 1 refresh token. What will happen if user will authenticate from multiple devices?

Reply:
This is not fixed.